### ISSUE TITLE
[Feature request] Show current working directory (cwd) in window title

### ISSUE THREAD

**Issue by probonopd** [lxqt/qterminal#1004]
**Is your feature request related to a problem? Please describe.**

I'm always frustrated when I have many instances of KWin open and don't know which is which from looking at the list of open windows in my window manager

**Describe the solution you'd like**

Show current working directory (cwd) in window titles, instead of "Shell No. 1"

**Comment by yan12125** [comment]
That is done by shell startup scripts. Do you use bash, zsh or something else?

**Comment by stefonarch** [comment]
> many instances of KWin open

KWin?

**Comment by probonopd** [comment]
> KWin?

Sorry, meant QTerminal of course. I am running it in KWin on X11.

When I run `xprop` and click on a QTerminal window, I get:

```
WM_NAME(STRING) = "Shell No. 1"
_NET_WM_NAME(UTF8_STRING) = "Shell No. 1"
```

> Here, it has always been so.

Am I supposed to see the current working directory there?

**Comment by yan12125** [comment]
> Am I supposed to see the current working directory there?

Yes, as long as your shell startup scripts work as intended. Which shell do you use?

**Comment by probonopd** [comment]
I am using `zsh` and `sh`. Do you happen to have an example shell startup script that is known working? Thanks!

**Comment by probonopd** [comment]
Thanks @yan12125. I meant `csh`, but I think I can figure that out.
